src/clients/ntp-client_proposal.rb

```diff
@@ -315,10 +300,13 @@ def AskUser
     def WriteNtpSettings(ntp_servers, ntp_server, run_service)
       ntp_servers = deep_copy(ntp_servers)
       NtpClient.modified = true
+      NtpClient.ntp_conf.clear_pools
       if ntp_servers != []
```

Comment:
btw
```
ntp_servers << [ntp_server] if !ntp_servers.empty?
```
should work the same and drop the strange if-else branch

Reply:
hmm, I do not get your code, but probably something like `ntp_servers << ntp_server if ntp_servers.empty?` will work.

Reply:
hm yes that's it  ... square brackets were bonus for free :-/